src/lib/pages/code-details/index.tsx

```diff
@@ -62,8 +73,12 @@ const CodeDetailsBody = observer(
       publicProject,
       lcdCodeData: { codeHash, isLcdCodeLoading, isLcdCodeError },
     } = codeDataState;
+    const { getSchemaByCodeHash } = useSchemaStore();
+    const jsonSchema = codeHash ? getSchemaByCodeHash(codeHash) : undefined;
     const isMobile = useMobile();
 
+    const [tabIndex, setTabIndex] = useState(0);
```

Comment:
Consider using an enum type instead of an integer value. This change will enhance readability.